Editor/Scripts/Utilities/MeshSyncObserver.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using UnityEditor;
+
+namespace Unity.MeshSync.Editor.Analytics {
+    
+
+    /// <summary>
+    /// Hacky way to restore observers to MeshSync servers in scene
+    /// </summary>
+    [InitializeOnLoad]
+    public static class MeshSyncObserverStartUp {
+
+        private static void update() {
```

Comment:
Do you need to unsubscribe from EditorApplication.update here? if so, would it better to name the function init instead of update?

Reply:
I didn't unsubscribe in this method because I'm not certain that the whole scene is loaded at the same time, is that guaranteed?

Reply:
I think this has been addressed now